### PR TITLE
Unicredit PSU-Id-Type header mapping

### PR DESCRIPTION
**Pull request by n0rdy**

### CODE REVIEW

File: adapters/unicredit-adapter/src/main/java/de/adorsys/xs2a/adapter/service/impl/UnicreditAccountInformationService.java [hunk 1]
```diff
@@ -13,13 +13,18 @@
...
 import java.util.Map;
+import java.util.Set;
 
 import static de.adorsys.xs2a.adapter.http.ResponseHandlers.jsonResponseHandler;
 
 public class UnicreditAccountInformationService extends BaseAccountInformationService {
     private static final String AUTHENTICATION_CURRENT_NUMBER_QUERY_PARAM = "authenticationCurrentNumber";
     private static final String DEFAULT_PSU_IP_ADDRESS = "0.0.0.0";
+    private static final Set<String> POSSIBLE_PSU_ID_TYPE_VALUES = new HashSet<>(Arrays.asList("HVB_ONLINEBANKING", "UCEBANKINGGLOBAL"));
+    private static final String DEFAULT_PSU_ID_TYPE = "HVB_ONLINEBANKING";
```
Comment (spetrychenko) [inline adapters/unicredit-adapter/src/main/java/de/adorsys/xs2a/adapter/service/impl/UnicreditAccountInformationService.java]: could you put this default to the possible list

Comment (n0rdy) [reply adapters/unicredit-adapter/src/main/java/de/adorsys/xs2a/adapter/service/impl/UnicreditAccountInformationService.java]: Done, thanks

**Comment by berezkin88** [comment]
One thing concerning me here is that, whether one small change in one adapter should trigger changing the BaseAccountInformationService and the AbstractService?

**Comment by n0rdy** [comment]
> One thing concerning me here is that, whether one small change in one adapter should trigger changing the BaseAccountInformationService and the AbstractService?

It seems for me that PSU-ID-Type is the header that is going to be changed from time to time (the same as PSU-ID), so I just implemented the same behavior for it. Anyway, any other adapter is being triggered by this change.
Also, I don't want to rewrite the entire createConsent() method for this small change only.

@spetrychenko , we need the third opinion here.

**Comment by spetrychenko** [comment]
@berezkin88 BaseAccountInformationService can evolve from time to time because it was designed against sandboxes. I agree, that the very first case also could be implemented inside the adapter.
Let me repeat words of one known classic: "it's ok for me"

**Pull request merged**